OrderNotificatorClient/src/app/app.module.ts

```diff
@@ -10,7 +10,7 @@ import { OrdersTableComponent } from './shared/orders-table/orders-table.compone
 import { TimerComponent } from './shared/timer/timer.component';
 import { TimeSelectorComponent } from './shared/time-selector/time-selector.component';
 import { PizzaComponent } from './pizza/pizza.component';
-
+import { OrderService } from './shared/service/order.service';
```

Comment:
Do we need that import?

Reply:
I thougth, that we should add OrderService to app.module.ts